packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -165,6 +165,178 @@ describeWithDOM('mount', () => {
       expect(() => wrapper.state('key')).to.throw('ReactWrapper::state("key") requires that `state` not be `null` or `undefined`');
     });
 
+    describeIf(is('>= 0.14'), 'wrappingComponent', () => {
```

Comment:
react 13 can't support this?

Reply:
We talked about this a bit way back in the early days of this PR (feels like ages ago haha) but the issue here is more just that this test suite is not capable of testing react 13 because of react 13's owner-based context API. This test suite requires parent-based context. (You'll notice that none of the `options.context` tests run on react 13 either.)

Reply:
ok, fair enough :-) we'll postpone that until after landing all the current context PRs.